test/CMakeLists.txt

```diff
@@ -1,9 +1,14 @@
-cmake_minimum_required(VERSION 3.7)
+cmake_minimum_required(VERSION 3.15)
+
+set(LIB_TARGET_UUID)
+if(NOT WIN32)
+    find_package(uuid REQUIRED)
+    set(LIB_TARGET_UUID uuid)
+endif()
```

Comment:
Note: On Windows, don't depend on libuuid since it doesn't exist. On Linux, keep using libuuid to validate interoperability.